fvt/storage/storage_test.go

```diff
@@ -130,11 +130,18 @@ var _ = Describe("ISVCs", func() {
 			//   from the old to the new pod
 
 			// make a shallow copy of default configmap (don't modify the DefaultConfig reference)
-			// keeping 1 pod per runtime and don't scale to 0
+			// we should keep 1 pod per runtime and don't scale to 0
```

Comment:
A bit confused by this comment, it states "don't scale to 0" but we want to enable it right below for resource constraint environments now. Could it be clarified?

Reply:
right, I meant to keep it that way. We really should not have to scale to zero, in general, but, for running the FVT on the GH Actions runner nodes with only 2 CPUs, we might have to scale to zero 🙃

Reply:
There are still timeouts happening in other places. Marking this PR as a draft for now.

Reply:
I removed the conflicting comment